usr/plugins/devel/caddy/src/opnsense/service/templates/Pischem/Caddy/Caddyfile

```diff
@@ -1,24 +1,27 @@
 # DO NOT EDIT THIS FILE -- OPNsense auto-generated file
 
+{% set generalSettings = helpers.getNodeByTag('Pischem.caddy.general') %}
+
 # Global Options
 {
     storage file_system {
         root /usr/local/etc/caddy
     }
     log {
+        {% if generalSettings.LogAccessPlain == "0" %}
```

Comment:
Same here as below, a default value would be a good choice for code resilience against errors.